.github/workflows/enforce-labels.yml

```diff
@@ -0,0 +1,13 @@
+name: Enforce PR labels
+
+on:
+  pull_request:
+    types: [labeled, unlabeled, opened, edited, synchronize]
+jobs:
+  enforce-label:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: yogevbd/enforce-label-action@2.1.0
+      with:
+        REQUIRED_LABELS_ANY: "client,runtime,breaksapi,ci,bug"
```

Comment:
Let's consider who is the primary consumer of this labels - in our case, right now, it's the CI for generating the release notes draft.

Which labels do we want to _enforce_ therefore? 
`client` and `runtime` look like definite candidates.

But for example, if a PR is only tagged with `breaksapi`, is that good enough?
Or just with a `bug`?

Reply:
> Let's consider who is the primary consumer of this labels - in our case, right now, it's the CI for generating the release notes draft.

Keeping this is mind I suggested the labels. We should add `xcm` also IMO. 
I think runtime, client, xcm and ci will cover everything. We can add `others` in case we missed anything and will keep on updating it as the scenarios emerge.

Reply:
What changes should result in `xcm` tag?

If I modify RPC tests, is that a `ci` tag? And if we modify the github workflow, is that also a `ci` tag?

Reply:
I hoped that `xcm` could contain things related to xcm-simulator, but that would complicate it.
Let's add `tests` which will include xcm-simulator tests and rpc-tests (which you don't need to track for release notes). Workflows should definitely be in `ci`.
Things like third-party and scripts could be in `other` or we can have labels by the same name respectively.

Reply:
That makes more sense to me :+1: 

Please also document this somewhere & share it with the team.